requirements-tests.txt

```diff
@@ -1,6 +1,8 @@
 wheel
 setuptools
 nose
```

Comment:
We need nose yet after this migration?

Reply:
Definitely not https://github.com/petl-developers/petl/pull/584/commits/9ff34359bc5d38f971943f526bc226a06f8d8cea